guake/main.py

```diff
@@ -240,17 +240,29 @@ def main():
     parser.add_argument(
         "--split-vertical",
         dest="split_vertical",
-        action="store_true",
-        default=False,
-        help=_("Split the selected tab vertically."),
+        metavar="SPLIT_PERCENTAGE",
+        action="store",
+        type=int,
+        nargs="?",
+        const=50,
+        default=None,
+        choices=range(101),
```

Comment:
The range needs to be constrained on the edges. Math error at choice = 0, and the behavior at 100 may be unintuitive, takes up the whole tab but still leaves the previous split in the tab hidden and crammed away. Trimming the range to 1-99 should mop these up, and maybe should be 5-95 but maybe there's someone out there with a giant screen who wants a 98% split, I guess 1-99 is fine.

Reply:
Good catch! I changed it to range(1,100), which limits the options to 1-99 for both horizontal and vertical..